### ISSUE TITLE
Build instructions wrong (default github instructions for cpuminer point to cpuminer-multi)

### ISSUE THREAD

**Issue by agentpatience** [zcoinofficial/cpuminer#14]
Please see that the instructions to build cpuminer + mtp is inaccurate in parts.
When someone arrives on https://github.com/zcoinofficial/cpuminer they get build instructions that are not accurate on the bottom of the project page (Download). The documentation needs to be updated and the links as well as per the instructions will only build a non-mtp supported build of cpuminer-multi but not cpuminer +mtp. 

Proper build instructions for Ubuntu 18.x:

sudo apt-get update -y
sudo apt-get upgrade -y
sudo reboot
git clone https://github.com/zcoinofficial/cpuminer.git
cd cpuminer/
chmod a+x configure
chmod a+x build.sh 
chmod a+x autogen.sh
sudo apt-get install automake autoconf pkg-config libcurl4-openssl-dev libjansson-dev libssl-dev libgmp-dev make g++
sudo apt-get install zlib1g-dev
  ./build.sh

Hope this helps!

**Comment by djm34** [comment]
build.sh isn't used. 
configure is autogenerated, so basically you don't need to do a chmod on it. 
autogen.sh doesn't need a chmod if you if you do a git clone. 
the package installation should be done doing a git clone. 

From this build, you build a cpuminer-multi with mtp support. The whole building instruction is exactly the same as the one used for "standard cpuminer-multi" the only difference is the absence of CFLAGS="-march=native" which isn't needed here

**Comment by agentpatience** [comment]
I don't understand completely -- in order for me to build from git clone I had to do the steps above, none of cloned files from git were executable so chmodding was required for me. I successfully used ./build.sh method?

**Comment by djm34** [comment]
yeah but installing the dependencies and various, definitely comes first...
